content/_data/logo/smurf.yml

```diff
@@ -0,0 +1,9 @@
+---
+name: 'Smurf'
+url: '/images/logos/smurf/smurf.png'
+url_256: '/images/logos/smurf/256.png'
+vector:
+credit: 'Kseniia Nenasheva'
+credit_url: 'https://twitter.com/ks_nenasheva'
+source: 'Belgium JAM'
```

Comment:
It is not an accepted Belgium JAM logo. @jmMeessen also had some legal concerns about it.
CC @ksenia-nenasheva